### ISSUE TITLE
There's a wrong assignment in the location list

### ISSUE THREAD

**Issue by TineUser** [grokability/snipe-it#13752]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

When loading the location list I see a wrong assignment: In the column "Actual Location" there are listed the assets that are assigned to the location and in the column of the belonging assets there are no values:

![image](https://github.com/snipe/snipe-it/assets/77736099/0cc0c4a2-c3a7-496a-8e9f-eaddc02ac614)

### Reproduction steps

Only opening the location list.

### Expected behavior

The column "Actual Location" doesn't make sense in this list. I think it can be deleted. And the assigned assets should be shown in the right column.

### Snipe-IT Version

6.2.2

### Operating System

Windows

### Web Server

Apache

### PHP Version

8.0.19

### Operating System

Windows

### Browser

Microsoft Edge (Chromium-based)

### Version

117.0.2045.60

**Comment by TineUser** [comment]
The bug isn't solved with Snipe-IT 6.2.3.

**Comment by snipe** [comment]
I don't actually see where the bug is. (It doesn't help that your screenshot isn't in English)

**Comment by TineUser** [comment]
Aktueller Standort --> Actual Location
Zugewiesene Assets --> Assigned Assets
Now it should be clear that there's something wrong with column title and the values shown in the column.

**Comment by TineUser** [comment]
If I understand this table correct then you see under actual location the asstes where the (default) location of the asset is set to the specified location but there aren't assigned to it. And under assigned assets you see the asstes which are explicit assigned to the specified location, right? This is a bit confusing.

**Comment by snipe** [comment]
Hi there - I know this is a pretty old issue, but I wanted to check and see if you ever got this resolved? If so, can you please close the ticket? If not, can you update us on what version of Snipe-IT you're currently using, so we can see if it's still an issue on latest?

:) Thanks!

**Comment by TineUser** [comment]
With version 8.2.1 - Build 19068 this issue is still there:

<img width="1127" height="99" alt="Image" src="https://github.com/user-attachments/assets/47a6ed5d-5954-4fd0-bdfb-22f08a2ec7f8" />

Aktueller Standort --> Actual Location
Zugewiesene Assets --> Assigned Assets

**Comment by snipe** [comment]
I'm not sure that's true. Actual location is tied into the location of the person/place/thing it's assigned to, where default location is where it lives when it's not currently checked out.

**Comment by TineUser** [comment]
All our assets are assigned to a location:

<img width="492" height="335" alt="Image" src="https://github.com/user-attachments/assets/a9c3f431-b2a5-4df6-b4ba-7d03d8048186" />

But this couldn't be seen in the location list.

**Comment by snipe** [comment]
I still don't think that's true. 

https://demo.snipeitapp.com/locations/1

There is no "actual" location. There's default location and then what we presume is the current location if it's checked out. 



<blockquote><img src="https://demo.snipeitapp.com/favicon.ico" width="48" align="right"><div><strong><a href="https://demo.snipeitapp.com/locations/1">Snipe-IT Asset Management Demo</a></strong></div></blockquote>

**Comment by TineUser** [comment]
The column in the list, "Aktueller Standort", means translated actual location. If default or standard location is meant with this column  then it should be translated to "Standard-Standort".

But why is the assigned assets column shown with "0" although the assets are assigned to the location - see 5th column in the screenshot above?

**Comment by snipe** [comment]
>  If default or standard location is meant with this column then it should be translated to "Standard-Standort".

I am not fluent in every language, unfortunately, so I have to trust the community to translate some of these strings for us. 

> But why is the assigned assets column shown with "0" although the assets are assigned to the location - see 5th column in the screenshot above?

I couldn't tell you without knowing more about what your data looks like. 

That first column, assigned assets (you can see that when you mouse over on the icon) is the count of assets that are assigned to that location. The house with the flag is the "home" or default location of the assets.

<img width="171" height="104" alt="Image" src="https://github.com/user-attachments/assets/d077e64e-306e-46a7-9e7e-e3ed0d14f1da" />
<img width="186" height="93" alt="Image" src="https://github.com/user-attachments/assets/f4979636-2000-4551-b512-93e05fa7e19b" />

You can see how this behavior works on the demo.

**Comment by TineUser** [comment]
So there's a difference between your screenshots an mine:

<img width="1199" height="90" alt="Image" src="https://github.com/user-attachments/assets/bee7e584-92d3-4c9c-a77f-d56abb7921eb" />

I don't se an icon for the default location. I only have a column named "Aktueller Standort" which probably means the default location.

What do you need to konw about my data to find the issue why te assigned assets are shown with "0"?

**Comment by snipe** [comment]
That column might be hidden - use the column selector and see if you can expose it.

**Comment by TineUser** [comment]
Yes, you're right, the default lacation column was hidden. I now activated the column to be shown:

<img width="1136" height="85" alt="Image" src="https://github.com/user-attachments/assets/f166a7d5-5747-4895-93e7-7474233d5436" />

So I don't understand the column "Aktueller Standort" which doesn't seem to be the default location as expected and shows other values than the default location column.

**Comment by snipe** [comment]
According to the translation files, "Aktueller Standort" is the translation for "Current location", which via the Presenter is pulling:

```php
public function assets()
    {
        return $this->hasMany(\App\Models\Asset::class, 'location_id')
            ->whereHas(
                'assetstatus', function ($query) {
                    $query->where('status_labels.deployable', '=', 1)
                        ->orWhere('status_labels.pending', '=', 1)
                        ->orWhere('status_labels.archived', '=', 0);
                }
            );
    }
```

so it's looking for all assets that have that location as their `location_id` and are not archived.

**Comment by TineUser** [comment]
Going into the location I see the following values:

<img width="362" height="112" alt="Image" src="https://github.com/user-attachments/assets/5862d2f8-f1cf-4946-b4dd-9382a92ea2f7" />

There you can see that the value of "Aktueller Standort" is shown with the icon of the assigned assets.

So why do we have a column "Aktueller Standort" (current location) which should be the assigned assets to the location?

**Comment by snipe** [comment]
Do you by any chance have a lot of archived assets?

**Comment by snipe** [comment]
I've opened up a new issue about this. I see what's going on.

**Comment by snipe** [comment]
In the meantime, if you want those numbers to agree, uncheck the "show archived in lists" checkbox in the admin settings > general.
